corehq/apps/export/models/new.py

```diff
@@ -1616,7 +1616,8 @@ def get_value(self, domain, doc_id, doc, base_path, split_column=False, **kwargs
             return value
 
         if not isinstance(value, basestring):
-            return [None] * len(self.item.options) + [] if self.ignore_unspecified_options else [value]
+            unspecified_options = [] if self.ignore_unspecified_options else [value]
+            return [None] * len(self.item.options) + unspecified_options
```

Comment:
nice do you think you could add a minor test for this? https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/export/tests/test_export_column.py#L39